src/Plugin.php

```diff
@@ -219,4 +233,48 @@ private function utf8_chr($cp) {
 
         return json_decode('"\u'.bin2hex(pack('n', $lead)).'\u'.bin2hex(pack('n', $trail)).'"');
     }
+
+    /**
+     * Switch statement to flip the special and punctuation characters
+     *
+     * @param string $char
+     *
+     * @return string
+     */
+    private function special_char($char)
```

Comment:
PSR-1 section 4.3 states: "Method names MUST be declared in camelCase()."

(Please correct other methods as well.)